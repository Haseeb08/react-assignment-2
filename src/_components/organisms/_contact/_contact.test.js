import React from "react";
import { create } from "react-test-renderer";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Contact from "./_contact";
import ContactForm from "../_contact-form/_contact-form";
import Title from "../../atoms/_title/_title";

jest.mock("../_contact-form/_contact-form", () => ({
    __esModule: true,
    default: ()=><div>I am mocked</div>
}) );

describe("Contact component testing", () => {
    

  test("renders", () => {
    const { container } = render(<Contact />);
    expect(container.textContent).toMatch("I am mocked");
  });

  test("Matches the snapshot", () => {
    const input = create(<Contact />);
    expect(input.toJSON()).toMatchSnapshot();
  });
});
