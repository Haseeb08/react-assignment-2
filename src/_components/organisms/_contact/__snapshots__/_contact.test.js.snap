// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component testing Matches the snapshot 1`] = `
<div
  className="makeStyles-main-3"
>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    You can reach out to me by filling the following form.
  </h2>
  <div
    className="makeStyles-form-4"
  >
    <div>
      I am mocked
    </div>
  </div>
</div>
`;
